### PR TITLE
Cleanup for 1.3.0

### PR DESCRIPTION
**Pull request by Thovi98**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Thovi98** [comment]
!testme

**Comment by Thovi98** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -33,6 +33,10 @@ ynh_systemd_action --action="stop" --service_name="${app}-worker" --log_path="sy
 if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
 	ynh_script_progression --message="Upgrading source files..." --weight=1
+	
+	# Remove the old files
+	ynh_secure_remove --file="$install_dir/api"
+	ynh_secure_remove --file="$install_dir/api"
```
Comment (Tagadda) [inline scripts/upgrade:39]:
```suggestion
```
duplicate lines :P

File: conf/funkwhale-server.service [hunk 2]
```diff
@@ -5,10 +5,19 @@ PartOf=__APP__.target
 
 [Service]
 User=__APP__
-Group=__APP__
```
Comment (Tagadda) [inline conf/funkwhale-server.service:8]: Not sure why you'd remove the group here and in others services ?

Comment (Thovi98) [reply conf/funkwhale-server.service:8]: Thank you!

**Comment by Thovi98** [comment]
!testme

**Pull request merged**